### ISSUE TITLE
(WCAG 2.1) Text is too faint in date picker

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3568]
Required to meet WCAG 2.1 guidelines. Identified during an accessibility audit against WCAG 2.1.

**WCAG SC**: [1.4.3. Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum)

**Observed results**: Some text in the calendar or datepicker component are faint. Does not meet minimum contrast ratio of 4.5:1. 

First screenshot:
- CrOS:
- Device(s) (brand, OS version, test key or release key): Version 107.0.7283.0 (Official Build) dev (64-bit)
- Screen reader name and version: Chromevox

Second screenshot:
- VoiceOver with Version 105.0.5195.125 on Mac OS 12.6
- ChromeVox with Version 105.0.5195. on Chromebook

**How to fix**: "Ensure that non-decorative text has a minimum color contrast ratio of 4.5:1 (foreground against background)." - a11y team

My notes: This is the built in browser date picker. Can we add custom styling to it? Will need more investigation.

![image](https://user-images.githubusercontent.com/9094240/192912200-aa59ac35-fd57-46df-acf2-98266956bcb9.png)
![image](https://user-images.githubusercontent.com/9094240/193164813-c53fe5eb-dfb6-471b-9576-45fa59eb4021.png)

**Comment by melanie-exygy** [comment]
This conversation moved to [Slack](https://civiform.slack.com/archives/C01R3BWAL1E/p1689702351389059)

**Comment by gwendolyngoetz** [comment]
Firefox example 
![image](https://github.com/civiform/civiform/assets/195162/ee90e666-894a-4fcf-8af8-91ecceb335b9)

**Comment by gwendolyngoetz** [comment]
Android example

![android](https://github.com/civiform/civiform/assets/195162/3c7df812-25ca-4313-accc-b5d792c7f245)

**Comment by melanie-exygy** [comment]
Discussed in Eng Sync meeting on 08/28/23 and decided to close this issue because changing the date picker would be too much effort and we would risk losing some of the features of the current date picker.
